pages/user guides/assessment_projects.md

```diff
@@ -40,6 +40,30 @@ You will then be brought to a new page that allows you to set out some goals for
 
 It's OK to skip the fields you are unsure about. You can always come back to this page and adjust the details as things evolve.
 
+## Bulk assign applications to a project
+You also have the ability to bulk assign the applications you selected to a project. Simply navigate to the `Applications` tab under `ASSESS` menu or manually visit `https://<your-subdomain>.tidalmg.com/#/apps`, select the applications you want to assign by click the checkboxes and then click the `Project Assign` button.
+
+{% include image.html file="projects_10.png" %}
+
+This will brings up a modal which contains a list of projects. You can search this list by name of the project by filtering the project name in the search bar above the list.
+
+{% include image.html file="projects_11.png" %}
+
+Additionally, you can easily create a brand new project by simply typing the project name in the search bar and clicking `Create New Project: [Your project name]`. A new project with the name you entered will be created within seconds and you can edit project details later in the project detail view.
```

Comment:
```suggestion
Additionally, you can easily create a brand new project by simply typing the project name in the search bar and clicking `Create New Project: [Your project name]`. Within seconds this will create a new project with the name you entered will be, and you can edit project details later in the project detail view.
```